inc/options/minify.php

```diff
@@ -81,504 +159,764 @@
+																	<option value="include-footer" <?php selected( $js_location, 'include-footer' ); ?>><?php esc_html_e( 'Embed before &lt;/body&gt;', 'w3-total-cache' ); ?></option>
+																</select>
+															</td>
+															<td>
+																<input class="js_file_delete js_enabled button" type="button" value="<?php esc_html_e( 'Delete', 'w3-total-cache' ); ?>" />
+																<input class="js_file_verify js_enabled button" type="button" value="<?php esc_html_e( 'Verify URI', 'w3-total-cache' ); ?>" />
+															</td>
+														</tr>
+													</table>
+												</li>
+												<?php
+											endforeach;
+										endif;
+									endforeach;
+								endforeach;
+							endif;
+						endforeach;
+						?>
+						</ul>
+						<div id="js_files_empty" class="w3tc-empty" style="display: none;">
+							<?php
+							echo wp_kses(
+								sprintf(
+									// translators: 1 opening HTML acronym tag, 2 closing HTML acronym tag.
+									__(
+										'No %1$sJS%2$s files added',
+										'w3-total-cache'
+									),
+									'<acronym title="' . __( 'JavaScript', 'w3-total-cache' ) . '">',
+									'</acronym>'
+								),
+								array(
+									'acronym' => array(
+										'title' => array(),
+									),
+								)
+							);
+							?>
+						</div>
+						<input id="js_file_add" class="js_enabled button" type="button" value="<?php esc_html_e( 'Add a script', 'w3-total-cache' ); ?>" />
+					</td>
+				</tr>
 			<?php endif; ?>
 			<?php
-Util_Ui::config_item( array(
-		'key' => 'minify.js.http2push',
-		'label' => '<acronym title="Hypertext Markup Language">HTTP</acronym>/2 push',
-		'control' => 'checkbox',
-		'checkbox_label' => __( 'Enable', 'w3-total-cache' ),
-		'description' => __( 'For better performance, send files to browser before they are requested when using the <acronym title="Hypertext Transfer Protocol">HTTP</acronym>/2 protocol.',
-			'w3-total-cache' ) .
-			( $this->_config->get_string( 'pgcache.engine' ) != 'file_generic' ? '' :
-				__( ' <br /><b>Not supported by "Disk: Enhanced" page cache method for Nginx</b>', 'w3-total-cache' ) )
-	) ); ?>
+			Util_Ui::config_item(
+				array(
+					'key'            => 'minify.js.http2push',
+					'label'          => '<acronym title="Hypertext Markup Language">HTTP</acronym>/2 push',
+					'control'        => 'checkbox',
+					'checkbox_label' => __( 'Enable', 'w3-total-cache' ),
+					'description'    => __( 'For better performance, send files to browser before they are requested when using the <acronym title="Hypertext Transfer Protocol">HTTP</acronym>/2 protocol.', 'w3-total-cache' ) .
+						(
+							'file_generic' !== $this->_config->get_string( 'pgcache.engine' ) ?
+							'' :
+							sprintf(
+								__(
+									' %1Not supported by "Disk: Enhanced" page cache method for Nginx%2',
```

Comment:
PHP fatal.
Invalid placeholders -- Change `%1` to `%1$s` and `%2` to `%2$s`.

Reply:
Fixed placeholders